Assets/LeapMotion/Core/Scripts/Animation/CubicHermiteSpline/HermiteSpline3.cs

```diff
@@ -95,16 +95,21 @@ public struct HermiteSpline3 {
     /// The time is clamped within the t0 - t1 range.
     /// </summary>
     public Vector3 PositionAt(float t) {
-      float i = Mathf.Clamp01((t - t0) / (t1 - t0));
-      float i2 = i * i;
-      float i3 = i2 * i;
-
-      Vector3 h00 = (2 * i3 - 3 * i2 + 1) * pos0;
-      Vector3 h10 = (i3 - 2 * i2 + i) * (t1 - t0) * vel0;
-      Vector3 h01 = (-2 * i3 + 3 * i2) * pos1;
-      Vector3 h11 = (i3 - i2) * (t1 - t0) * vel1;
-
-      return h00 + h10 + h01 + h11;
+      if (t > t1) {
```

Comment:
If you read the summary for this class, it says "t is clamped in the t0-t1 range". Not sure if you added this or if I did.. but we should make sure the summary tag isn't lying, either remove this part, or change the summary to say that t > t1 will extrapolate the evaluated point out linearly using the velocity at t1.